compiler-plugin-tests/src/test/resources/codeaction/add_interceptor_method/result1.bal

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/http;
+
+service class RequestInterceptor {
+	*http:RequestInterceptor;
```

Comment:
indentation issue?

Reply:
Added a comment to the file in [36a803d](https://github.com/ballerina-platform/module-ballerina-http/pull/992/commits/36a803d3b71ae67a41c896c3358f071f9f13a24e)